test/test.sh

```diff
@@ -931,6 +933,40 @@ test_password_server() {
 	return 0
 }
 
+# test password authentication from PgBouncer to PostgreSQL server using
+# auth_query to retrieve user's shadow password from pg_shadow
+test_shadow_password_server_login() {
```

Comment:
These test cases are reproducible examples of bug: https://github.com/pgbouncer/pgbouncer/issues/484